TestsFoundation/TestsFoundation/Fixtures/API/APICourseFixture.swift

```diff
@@ -59,7 +59,7 @@ extension APICourse {
             term: term,
             permissions: permissions,
             access_restricted_by_date: access_restricted_by_date,
-            image_download_url: image_download_url,
+            image_download_url: image_download_url?.absoluteString,
```

Comment:
Can you instead make the parameter a `String?` instead of a `URL?` to match the type?